addons/settings/gui_createMenu_color.sqf

```diff
@@ -98,6 +103,11 @@ for "_index" from 0 to (((count _defaultValue max 3) min 4) - 1) do {
         _color set [_index, _value];
         _ctrlSettingPreview ctrlSetBackgroundColor _color;
         SET_TEMP_NAMESPACE_VALUE(_setting,_currentValue,_source);
+
+        //If new setting is same as default value, grey out the default button
+        (_control getVariable QGVAR(linkedControls)) params ["", "", "_defaultControl"];
```

Comment:
OK. IDK why, but it keeps messing up the line of this ^  comment. Meant for L108.